ts/screens/modal/IdentificationModal.tsx

```diff
@@ -60,6 +61,7 @@ import {
   IdentificationInstructionsComponent,
   getBiometryIconName
 } from "../../utils/identification";
+import { MIN_HEIGHT_TO_SHOW_FULL_RENDER } from "../authentication/OptInScreen";
```

Comment:
What about extracting this constant to a separate utility file? What do you think?